src/menu/story.tsx

```diff
@@ -84,6 +84,13 @@ const menuItems = [
     leftAddon: <CrossDiscIcon />,
     href: 'menu-item-10',
   },
+  {
+    id: 'menu-item-11',
+    label: 'Menu item with onClick',
+    onClick: () => {
+      window.alert('Item clicked')
```

Comment:
Maybe you could use the Storybook `action` plugin here :)

Reply:
Yes e.g. `onClick={action('onClick')}`and it will appear in the Action Logger tab in storybook

Reply:
Done. Thanks for the tip!